test/conformance/gateway_conformance_test.go

```diff
@@ -24,12 +24,25 @@ import (
 var skippedTestsForTraditionalRoutes = []string{
 	// core conformance
 	tests.HTTPRouteHeaderMatching.ShortName,
+	// There is an issue with KIC when processing this scenario.
+	// TODO: https://github.com/Kong/kubernetes-ingress-controller/issues/6136
+	tests.GRPCRouteListenerHostnameMatching.ShortName,
+	// tests.GRPCRouteHeaderMatching.ShortName and tests.GRPCExactMethodMatching.ShortName may
+	// have some conflicts, skipping either one will still pass normally.
+	tests.GRPCExactMethodMatching.ShortName,
+}
+
+var skippedTestsForExpressionRoutes = []string{
+	// When processingvthis scenario,
+	// the expression router of Kong currently needs to add `priority` to the route.
```

Comment:
This comment doesn't explain why we skip the test. Is it because the priority does not work for some reason? Do we have an issue reference for this?

```suggestion
	// When processing this scenario, the Kong's expression router currently needs to
	// add `priority` to the route.
```

Reply:
The reason is that Kong's expression router needs to **manually** add priority to route requests to the correct backend service correctly. However, as you know, this is the conformance test and we cannot intervene in it or set priorities for it. So we can only skip this test case to avoid errors.


I have added a comment here https://github.com/Kong/kubernetes-ingress-controller/pull/5776#discussion_r1627346136

I think it may not be appropriate to include internal Slack links in the comments, so I did not paste them in the comments.

Reply:
Thanks for the explanation. 🙇